test/test_models.py

```diff
@@ -97,3 +98,19 @@ def test_add_stage_too_far_ahead(sample_with_stages):
     (stages, token) = models.get_sample_stages(s.obfuscated_id)
     assert 2 == len(stages)
     assert h.encode(2) == token
+
+
+def test_create_first_stage_file(storepath, sample_with_stages):
+    stage = sample_with_stages['stages'][0]
+    ssf = models.SampleStageFile(stage)
+    assert '00001/00001/00001-00000' == ssf.relative_file_path
```

Comment:
Actually, no.  SampleStageFile incorrectly uses only the 'method' and omits the 'stage' when constructing the path.  A future commit will make this clearer by including the resource name in the path directories and will fix the problem by adding the stage ID to the 'stage' component of the path.

At this point though I just wanted to get a test case in place.